### ISSUE TITLE
Place schema violations below constraint violations, add a note that schema violations are not currently enforced

### ISSUE THREAD

**Issue by matthew-morris-cessda** [cessda/cessda.cmv.server#122]
This is to reduce confusion regarding recent changes to the behavior of the CDC validator (see cessda/cessda.cdc.versions#545).

**Comment by matthew-morris-cessda** [comment]
@john-shepherdson Current test appearance

![image](https://github.com/cessda/cessda.cmv.server/assets/68543725/f933a38d-004a-4ef7-94b3-9a5953f22488)

**Comment by john-shepherdson** [comment]
Not sure it is clear what 'Not currently enforced' applies to. Also, concept of 'enforcement' is not clear. 

Need a statement at top of results that says something like:
Note that records with constrain violations will be visible in the CESSDA Data Catalogue (CDC) User Interface, but will be excluded from the OAI-PMH endpoint. Records with XSD Schema violations will be visible in the CDC User Interface and, for the time being, also included in the OAI-PMH endpoint.

**Comment by matthew-morris-cessda** [comment]
Seems a bit wordy to me. I do agree that "Not currently enforced" is unclear though.

**Comment by john-shepherdson** [comment]
Note that records with constrain violations are excluded from the CESSDA Data Catalogue OAI-PMH endpoint.
